save-demo/src/main/kotlin/com/saveourtool/save/demo/storage/ToolStorage.kt

```diff
@@ -14,9 +17,29 @@ private const val TOOLS_PATH = "tools"
  */
 @Component
 class ToolStorage(
-    private val configProperties: ConfigProperties,
+    configProperties: ConfigProperties,
 ) : AbstractFileBasedStorage<ToolKey>(Path.of(configProperties.fileStorage.location) / TOOLS_PATH) {
-    private val rootPath = Path.of(configProperties.fileStorage.location) / TOOLS_PATH
+    /**
+     * Todo: Implement DownloadToolService
+     */
+    init {
+        val supportedTools = listOf(
+            ToolKey("diktat", "1.2.3", "diktat"),
```

Comment:
If I understand the code correctly, the 1st and the 2nd argument can be extracted into a variable, e.g.:

```suggestion
            val diktat = "diktat" to "1.2.3"
            ToolKey(diktat.first, diktat.second, "diktat"),
```

Reply:
Well, yes, but is it really needed here and now?